src/v/cluster/feature_manager.cc

```diff
@@ -309,17 +309,36 @@ void feature_manager::verify_enterprise_license() {
     }
 
     const auto& license = _feature_table.local().get_license();
-    auto license_missing_or_expired = !license || license->is_expired();
+    std::optional<security::license> fallback_license = std::nullopt;
+    auto fallback_license_str = std::getenv("RP_FALLBACK_ENTERPRISE_LICENSE");
```

Comment:
I don't think there's tons of precedent for this, but maybe

```suggestion
    auto fallback_license_str = std::getenv("REDPANDA_FALLBACK_ENTERPRISE_LICENSE");
```

Reply:
The prior art of existing env vars:
* Redpanda:
```
RP_SI_CREDS_API_ADDRESS (undocumented)
RP_BOOTSTRAP_USER (only documented Redpanda env var)
RP_FIXTURE_ENV (testing)

REDPANDA_ENVIRONMENT (undocumented, set by operator to "kubernetes", used for phone-home data)

__REDPANDA_SKIP_IAM_TOKEN (testing)
__REDPANDA_LICENSE_CHECK_INTERVAL_SEC (testing)
__REDPANDA_TOPIC_REC_DL_CHECK_MILLIS (testing)
__REDPANDA_TEST_FEATURES (testing)
__REDPANDA_TEST_FEATURE_NO_AUTO_ACTIVATE_BRAVO (testing)
__REDPANDA_LATEST_LOGICAL_VERSION (testing)
__REDPANDA_EARLIEST_LOGICAL_VERSION (testing)
__REDPANDA_TEST_DISK_SIZE (testing)
```
* Rpk uses `REDPANDA_` prefix
* Wasm transforms use the `REDPANDA_` prefix

So, there is no clear consistency so far, but I think it makes sense to prefer the `REDPANDA_` prefix for this and env vars going forward. Some env vars also use the `__` prefix, but that seems to be the pattern for test-only variables, so I think we don't want to add that for this variable.

I've changed this to `REDPANDA_FALLBACK_ENTERPRISE_LICENSE` now.

Reply:
oh yeah, missed `RP_BOOTSTRAP_USER`. The `__` prefix is definitely meant for "hidden" knobs, so no worries there. I don't think it's worth belaboring too much, but we might want to ask product.